src/app/redux/UserSaga.js

```diff
@@ -67,6 +68,11 @@ export function* watchRemoveHighSecurityKeys() {
     yield* takeLatest('@@router/LOCATION_CHANGE', removeHighSecurityKeys);
 }
 
+function* getRecentPrice() {
+    const feed = yield call([api, api.getFeedHistoryAysnc]);
+    if (feed) yield put(userActions.setPriceFeed({price: feed[feed.length - 1]}));
```

Comment:
`getFeedHistoryAsync` returns an object with the feed under the `price_history` key